aspnetcore/toc.yml

```diff
@@ -804,7 +804,7 @@
     - name: IIS
       displayName: deploy, publish
       items:
-        - name: Overview
+        - name: Host and deploy on IIS
```

Comment:
Is this a one-off, or is there new guidance for naming the first topic of a node?

Reply:
Arg, reverted that. I read it wrong.